packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -4,167 +4,508 @@
+          surface: surface ?? Color(palette.neutral.get(99)),
+          onSurface: onSurface ?? Color(palette.neutral.get(0)),
+          surfaceVariant: surfaceVariant ?? Color(palette.neutralVariant.get(90)),
+          onSurfaceVariant: onSurfaceVariant ?? Color(palette.neutralVariant.get(30)),
+          inverseSurface: inverseSurface ?? Color(palette.neutral.get(20)),
+          onInverseSurface: onInverseSurface ?? Color(palette.neutral.get(95)),
+          inversePrimary: inversePrimary ?? Color(palette.primary.get(80)),
+          shadow: shadow ?? Color(palette.neutral.get(0)),
+          brightness: brightness,
+        );
+
+      case Brightness.dark:
+        return ColorScheme(
+          primary: primary ?? Color(palette.primary.get(80)),
+          onPrimary: onPrimary ?? Color(palette.primary.get(20)),
+          primaryContainer: primaryContainer ?? Color(palette.primary.get(70)),
+          onPrimaryContainer: onPrimaryContainer ?? Color(palette.primary.get(10)),
+          secondary: secondary ?? Color(palette.secondary.get(80)),
+          onSecondary: onSecondary ?? Color(palette.secondary.get(20)),
+          secondaryContainer: secondaryContainer ?? Color(palette.secondary.get(70)),
+          onSecondaryContainer: onSecondaryContainer ?? Color(palette.secondary.get(10)),
+          tertiary: tertiary ?? Color(palette.tertiary.get(80)),
+          onTertiary: onTertiary ?? Color(palette.tertiary.get(20)),
+          tertiaryContainer: tertiaryContainer ?? Color(palette.tertiary.get(70)),
+          onTertiaryContainer: onTertiaryContainer ?? Color(palette.tertiary.get(10)),
+          error: error ?? Color(palette.error.get(80)),
+          onError: onError ?? Color(palette.error.get(20)),
+          errorContainer: errorContainer ?? Color(palette.error.get(70)),
+          onErrorContainer: onErrorContainer ?? Color(palette.error.get(10)),
+          outline: outline ?? Color(palette.neutralVariant.get(60)),
+          background: background ?? Color(palette.neutral.get(10)),
+          onBackground: onBackground ?? Color(palette.neutral.get(90)),
+          surface: surface ?? Color(palette.neutral.get(10)),
+          onSurface: onSurface ?? Color(palette.neutral.get(100)),
+          surfaceVariant: surfaceVariant ?? Color(palette.neutralVariant.get(30)),
+          onSurfaceVariant: onSurfaceVariant ?? Color(palette.neutralVariant.get(80)),
+          inverseSurface: inverseSurface ?? Color(palette.neutral.get(90)),
+          onInverseSurface: onInverseSurface ?? Color(palette.neutral.get(20)),
+          inversePrimary: inversePrimary ?? Color(palette.primary.get(40)),
+          shadow: shadow ?? Color(palette.neutral.get(0)),
+          brightness: brightness,
+        );
+    }
+  }
 
   /// Create a ColorScheme based on a purple primary color that matches the
   /// [baseline Material color scheme](https://material.io/design/color/the-color-system.html#color-theme-creation).
   const ColorScheme.light({
```

Comment:
As with the basic constructor, best to refer to ColorScheme.fromSeed() here.

Reply:
Not sure I get why here. If they are looking for a pre-built 'light' color scheme, why would they want to use `fromSeed` (other than to make sure they get a full color scheme), and what should they use for the seed color? This kind of points out the problem with not having a M3 version of this and `dark`.

Reply:
I was thinking that it would be helpful to get apps off of ColorScheme.light(), since there isn't a perfect migration path that preserves the existing colors and adds new ones are comparable to what you'd get from ColorScheme.fromSeed().  Hopefully we can migrate developers that have used ColorScheme.light() to something like ColorScheme.fromSeed(seedColor: const Color(_just the right purple seed color here_). So for now we just remind them that the fromSeed factory is almost as trivial to write and so much more useful.